var/spack/repos/builtin/packages/adiak/package.py

```diff
@@ -12,10 +12,17 @@ class Adiak(CMakePackage):
 
     homepage = "https://github.com/LLNL/Adiak"
     url      = "https://github.com/LLNL/Adiak/releases/download/v0.1/adiak-v0.1.1.tar.gz"
+    git      = "https://github.com/LLNL/Adiak"
+
+    maintainers = ["daboehme", "mplegendre"]
 
     variant('mpi', default=True, description='Build with MPI support')
     variant('shared', default=True, description='Build dynamic libraries')
 
+    version('0.3.0-alpha', commit='054d2693a977ed0e1f16c665b4966bb90924779e',
+            submodules=True)
+    version('0.2.1', commit='950e3bfb91519ecb7b7ee7fa3063bfab23c0e2c9',
```

Comment:
You might want to set `preferred=True` here

Reply:
Done